tests/lib/c_lib/thrd/prj.conf

```diff
@@ -4,3 +4,9 @@ CONFIG_TEST_USERSPACE=y
 CONFIG_ZTEST_FATAL_HOOK=y
 CONFIG_MINIMAL_LIBC_NON_REENTRANT_FUNCTIONS=y
```

Comment:
Nit: maybe it would be a bit nicer if these 2 where in the testcase configuration for minimal libc or an overlay?

Reply:
Maybe even in the testcase.yaml. Good suggestion.